IDEOorg/steps-bot
# #162900982 As a developer, I want up-to-date API documentation for the Bot

#### What does this PR do?
Creates API documentation for API endpoints

#### Description of Task to be completed?
- add swagger documentation
- provide relevant API request responses
- update README.md with a link to the API documentation

#### How should this be manually tested?
- $ git clone https://github.com/IDEOorg/steps-bot.git
- $ git checkout ch-document-api-162900982.
- $ npm install
- $ npm start
- visit `http:localhost:3002/api-docs`
- observe that you can see the API documentation

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#162900982](https://www.pivotaltracker.com/story/show/162900982)
#### Screenshots (if appropriate)
<img width="1680" alt="screenshot 2019-02-06 at 22 42 01" src="https://user-images.githubusercontent.com/20840601/52368859-77fff180-2a60-11e9-882a-f796d462d6ea.png">


#### Questions
N/A
